### PR TITLE
Bump

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://github.com/Unidata/netcdf-c/releases/tag/v4.4.1.1

### CODE REVIEW

**Comment by dopplershift** [comment]
I'm going to go out on a limb and say the compile failure is related to changes made to ncgen in 4.4.1.1. @WardF care to weigh in here? The failure line talks about `largealign` being undefined: https://ci.appveyor.com/project/conda-forge/libnetcdf-feedstock/build/1.0.26/job/t18wqts0ta04w19k#L899

**Comment by ocefpaf** [comment]
> I'm going to go out on a limb and say the compile failure is related to changes made to ncgen in 4.4.1.1. @WardF care to weigh in here? The failure line talks about `largealign` being undefined: https://ci.appveyor.com/project/conda-forge/libnetcdf-feedstock/build/1.0.26/job/t18wqts0ta04w19k#L899

Thanks for looking into it @dopplershift! BTW, should I add you as a maintainer here :wink:

File: appveyor.yml [hunk 1]
```diff
@@ -14,28 +12,18 @@ environment:
...
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda
```
Comment (ocefpaf) [inline appveyor.yml]: Please ready https://github.com/conda-forge/libnetcdf-feedstock/pull/8#issuecomment-262108343

**Comment by ocefpaf** [comment]
Closing this as my experimentation is broken now.

@jakirkham please read https://github.com/conda-forge/libnetcdf-feedstock/pull/8#issuecomment-262108343 and follow https://github.com/conda-forge/libnetcdf-feedstock/pull/4

File: appveyor.yml (hunk 1 above)
Comment (ocefpaf) [inline appveyor.yml]: > Ok, done. Hope that helps.

No it did not. The part you should read was:

> But this PR is not ready yet!

See #9 and please keep the noise level down as this is part of my day job.

**Comment by jakirkham** [comment]
Sorry for the trouble.

**Pull request closed without merging**